Teapot/Logger.swift

```diff
@@ -0,0 +1,106 @@
+import Foundation
+
+/// Wrapper for logs which can be toggled on and off to increase or decrease the amount of log barf you wish to see.
+public class Logger {
+
+    public enum LogLevel: Int {
+        case
+        // Logs all data sent or received to the console, including the string representation of the data which was sent or received.
+        incomingAndOutgoingData,
```

Comment:
I really prefer each case in its own line, instead of a mix of both declaring them all in one go, and breaking down lines. 😅

Reply:
Open a PR on the code standards then :trollface: